openhcl/underhill_attestation/src/igvm_attest/key_release.rs

```diff
@@ -35,6 +35,10 @@ pub(crate) enum KeyReleaseError {
 pub(crate) enum AkvKeyReleaseJwtError {
     #[error("invalid JWT format, data: {0}")]
     InvalidJwtFormat(String),
+    #[error("invalid JWT header format, data: {0}")]
```

Comment:
Should we include the raw data in these errors, instead of the lossy version? Doesn't that result in us losing the important bits (the invalid utf8)?